### PR TITLE
Dev options description

### LINKED ISSUE #133: Update description available on the plugin option menu
The current description in the option menu needs to be changed to include a new website URL [ https://earthdailyagro.com/geosys-api/]( https://earthdailyagro.com/geosys-api/) and replace the old one which points to [https://www.geosys.com/bridge/](https://www.geosys.com/bridge/).

A new button labeled `About` should be added in the option menu and it should open a dialog that shows following social media links

```
- EarthDaily agro https://earthdailyagro.com/
- News https://earthdailyagro.com/news/
- Careers https://earthdailyagro.com/careers/
- Contact https://earthdailyagro.com/contact/ 
- Twitter - https://twitter.com/EarthDailyAgro
- Linkedin - https://www.linkedin.com/company/115836/admin/
- Youtube https://www.youtube.com/channel/UCy4X-hM2xRK3oyC_xYKSG_g
```

suggested UI for the social media dialog

image

### PR DESCRIPTION
**Pull request by vermeulendivan**
Fixes #133 
Link on options dialog updated to make use of https://earthdailyagro.com/geosys-api/.
About button added to options dialog, which opens an about dialog. About shows links to important/informative sites.

image

### CODE REVIEW

File: geosys/utilities/about.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    :param message: An optional message object to display in the dialog.
+    :type message: Message.Message
+    """
+
+    help_path = mktemp('.html')
+    with open(help_path, 'wb+') as f:
+        help_html = get_about_html(message)
+        f.write(help_html.encode('utf8'))
+        path_with_protocol = 'file://' + help_path
+        QDesktopServices.openUrl(QUrl(path_with_protocol))
```
Comment (Samweli) [inline geosys/utilities/about.py:46]: @vermeulendivan does this work for all Operating systems?

**Review by Samweli — CHANGES_REQUESTED** [review]
This looks good.

Can we also update the plugin icon on the about page and  use the color scheme shown on the original issue, thanks.

**Comment by vermeulendivan** [comment]
@Samweli The icon and logo is updated in the https://github.com/GEOSYS/qgis-plugin/pull/142 PR, which will also update the about dialog's logo

File: geosys/utilities/about.py (hunk 1 above)
Comment (vermeulendivan) [reply geosys/utilities/about.py:46]: @Samweli That code section is part of the original code section. The plugin works perfect in Windows

**Review by Samweli — APPROVED** [review]

**Pull request merged**